### ISSUE TITLE
Wont compile deckercomments

### ISSUE THREAD

**Issue by getarun** [Suthek/ShadowTeXSR5#1]
Issue: 
```
\begin{deckercomment}{Suthek}
Look, mom, I'm in a rulebook! :D
\end{deckercomment}
```
throws error while compiling with 
```
xelatex Ruleboook.tex
```
Error: 
```
kpathsea: Running mktextfm Decker
/usr/share/texlive/texmf-dist/web2c/mktexnam: Could not map source abbreviation D for Decker.
/usr/share/texlive/texmf-dist/web2c/mktexnam: Need to update /usr/share/texlive/texmf-dist/fonts/map/fontname/special.map?
mktextfm: Running mf-nowin -progname=mf \mode:=ljfour; mag:=1; nonstopmode; input Decker
This is METAFONT, Version 2.7182818 (TeX Live 2019/dev/Debian) (preloaded base=mf)


kpathsea: Running mktexmf Decker
! I can't find file `Decker'.
<*> ...:=ljfour; mag:=1; nonstopmode; input Decker
                                                  
Please type another input file name
! Emergency stop.
<*> ...:=ljfour; mag:=1; nonstopmode; input Decker
                                                  
Transcript written on mfput.log.
grep: Decker.log: No such file or directory
mktextfm: `mf-nowin -progname=mf \mode:=ljfour; mag:=1; nonstopmode; input Decker' failed to make Decker.tfm.
kpathsea: Appending font creation commands to missfont.log.
Underfull \vbox (badness 4229) has occurred while \output is active

! Package fontspec Error: The font "Decker" cannot be found.

For immediate help type H <return>.
 ...                                              
                                                  
l.124 \end{deckercomment}
```

Minimal working example
```
\documentclass{ShadowTeXSR5}

\usepackage{lipsum}

\begin{document}

\chapter{This is a Chapter}
\lipsum

\section{Deckerchat}
\lipsum[1-5]

\begin{deckercomment}{Suthek}
	Look, mom, I'm in a rulebook! :D
\end{deckercomment}

\end{document}
```
Versions:
```
XeTeX 3.14159265-2.6-0.99999 (TeX Live 2019/dev/Debian)
kpathsea version 6.3.1/dev
Copyright 2018 SIL International, Jonathan Kew and Khaled Hosny.
There is NO warranty.  Redistribution of this software is
covered by the terms of both the XeTeX copyright and
the Lesser GNU General Public License.
For more information about these matters, see the file
named COPYING and the XeTeX source.
Primary author of XeTeX: Jonathan Kew.
Compiled with ICU version 63.1; using 63.1
Compiled with zlib version 1.2.11; using 1.2.11
Compiled with FreeType2 version 2.9.1; using 2.9.1
Compiled with Graphite2 version 1.3.13; using 1.3.13
Compiled with HarfBuzz version 2.3.1; using 2.3.1
Compiled with libpng version 1.6.36; using 1.6.36
Compiled with poppler version 0.68.0
Compiled with fontconfig version 2.13.1; using 2.13.1
```

**Comment by Suthek** [comment]
Aye, checked out the function and it used an outdated assignment from the font I had installed on my system instead of drawing from the font family defined by the files in the folder. Should be a quick fix.

**Comment by Suthek** [comment]
Fixed with 211d5156716d0f859b5cec71da639dc3e2bd60a9
